### DISCUSSION TITLE
未在类型“IconElement”中找到可附加的属性“Geometry”

### DISCUSSION THREAD

**Discussion by quzefeng** [HandyOrg/HandyControl#1511] [Q&A]
管理员您好；第一次使用这个框架，估计这个问题问的比较简单，但是确实为难了我好久；请帮助支持一下如何解决；1.我使用包管理器HandyControl安装了咱们的包；2.我的开发工具是VS2022;3.出现问题的是我对按钮使用controls:BorderElement.CornerRadius="15"，这样的属性，系统无法识别；我不知道该怎么办了，请求您的帮助
![111](https://github.com/HandyOrg/HandyControl/assets/17823304/9a9a9561-30f3-4820-9a00-d7f5620da90b)
![222](https://github.com/HandyOrg/HandyControl/assets/17823304/b529ecf4-790d-42b7-93ab-817ed028bf48)

![333](https://github.com/HandyOrg/HandyControl/assets/17823304/ae075c11-5e87-4b6e-820f-d6e94124e7b7)

**Comment by NaBian** [comment]
把 `controls:` 改成 `hc:`

**Reply by wangjunjx8868** [reply]
<hc:SideMenuItem.Icon>
      <TextBlock Text="&#xf30e;"/>
  </hc:SideMenuItem.Icon> 图标不显示
